backend/utils/other/storage.py

```
@@ -5,21 +5,66 @@
+            ('Syncing local', bucket_sync),
+            ('Backups', bucket_backups)
+        ]
+        
+        # Verify required buckets exist and are set
+        for bucket_name, bucket_id in required_buckets:
+            if not bucket_id:
+                raise EnvironmentError(f"Required bucket '{bucket_name}' environment variable is not set")
+            try:
+                client.get_bucket(bucket_id)
+            except Exception as e:
+                raise EnvironmentError(f"Error accessing required Google Cloud Storage bucket '{bucket_name}' ({bucket_id}): {str(e)}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)